src/katsdpcontroller/generator.py

```diff
@@ -1268,6 +1275,13 @@ def get_baseline_index(a1, a2):
                     "dst": f"{{endpoints_vector[multicast.{stream.name}_spead][{i}]}}",
                     "pol": stream.src_pol,
                 }
+                if stream.dither is not None:
+                    if stream.dither == "none":
+                        output_config["dither"] = "false"
+                    elif stream.dither == "uniform":
+                        output_config["dither"] = "true"
```

Comment:
As noted on the linked PR: it might be better to update fgpu to accept `uniform` / `none` and then this remapping code can go away.